server/grammar/YmlParser.g4

```diff
@@ -300,8 +300,10 @@ chainedCall:
     )*
 ;
 
+unnamedStaticDeclaration: className=ymlId (fieldValue)* SEMICOLON;
```

Comment:
The fact that the new rule is for “unnamed” declarations makes me wonder if I understood correctly the RN entry above.

Is it “Inline-named (object instances)”, “Inline (named object instances)”? Or something else entirely? :exploding_head:

Reply:
It is `Inline (named object instances)`. I can probably rename this rule or just integrate it directly in `inlineDeclaration`, since it will never be used elsewhere?

Reply:
Hum my question was, like, is it

named inline
vs.
unnamed inline

or

named inline
vs.
named not inline

I think your original RN entry hints towards the latter, while the code here does the opposite.

Pfiou, I think I finally succeeded in putting into words what’s been nagging me since the beginning.

Reply:
(and regardless: no I think keeping separate rules, kinda like with functions, will make things clearer.)

Reply:
oh but you removed it already lol. Do as you wish for that, I guess.

Reply:
We try in this PR to make public the instances defined as inline and having a name. So, we're more on a

inline and named
vs.
inline but unnamed

Reply:
I removed the new rule, anyway. So this won't be an issue anymore.

Reply:
I still needed that answer to make sure my sugg on the RN wasn’t nonsensical. Seems OK. I’ll let the others test, though.